### ISSUE TITLE
How do you associate the time the each data has been collected?

### ISSUE THREAD

**Issue by SleepEasyCenter** [viatom-develop/LepuDemo#1]
Is there no time associated with each data point collected from the O2 Ring?

only can get start time?

**Comment by Chenyongfeng-424** [comment]
Yes, there is no other way. If the data is collected in real time, the start collection time can be used to calculate the time of the corresponding point based on the sampling rate. If the data is downloaded from the device, the recording start time can be obtained, and then the time of the corresponding point can be calculated based on the sampling rate.

**Comment by Chenyongfeng-424** [comment]
Yes, there is no other way. If the data is collected in real time, the start collection time can be used to calculate the time of the corresponding point based on the sampling rate. If the data is downloaded from the device, the recording start time can be obtained, and then the time of the corresponding point can be calculated based on the sampling rate. @SleepEasyCenter
